include/class.mail.php

```diff
@@ -326,6 +326,14 @@ public function readLine() {
              return trim($result);
          }
 
+         public function login($user, $password, $tryApop = true) {
+             try {
+                parent::login($user, $password, $tryApop);
+                return true;
+             } catch (\Throwable $e) {
+                 throw new Exception(__('login failed').': '.$e->getMessage());
```

Comment:
update this later to just `$e->getMessage()`

Reply:
I wouldn't update this. $e-getMessage() returns only "last request failed". 
A normal User don't know, what this meens. So he know, It happens on login...